src/components/CurrentTime/CurrentTime.jsx

```diff
@@ -1,7 +1,22 @@
 import classes from './CurrentTime.module.scss';
+import { useState, useEffect } from 'react';
 
 function CurrentTime() {
-  return <div className={classes.currentTime}>17:00</div>;
+  const [currentTime, setCurrentTime] = useState(new Date().toTimeString());
```

Comment:
I guess you could benefit a little bit here from lazy initializing

Reply:
Done.